GitUI/CommandsDialogs/SettingsDialog/Pages/ScriptsSettingsPage.cs

```diff
@@ -20,6 +20,7 @@ public partial class ScriptsSettingsPage : SettingsPageWithHeader
 
 User Input:
 {UserInput}
+{UserInput:a popup label|a default value|}
```

Comment:
❓ more intuitive and less error-prone (missing trailing "|"):
```suggestion
{UserInput}
{UserInput:label}
{UserInput:label=default value}
```

Reply:
{UserInput:label=defaultValue}

Reply:
We can find a better syntax like maybe `{UserInput[label=default value]}` but I really like to keep a end character otherwise it makes the parsing a hell if you want to have the possibility to make the default value a parsable expression with variables (like the example given in the description where the filename default value is built using the branch name variable)

Reply:
> it makes the parsing a hell

implement once, use many times
Does it really matter whether the default value is terminated by "}" or "|"? If only the interior of "{...}" is parsed, the default value end matches "$".